libs/api/domains/document-provider/src/lib/models/DocumentTypes.model.ts

```diff
@@ -0,0 +1,16 @@
+import { Field, Int, ObjectType } from '@nestjs/graphql'
+
+@ObjectType()
+export class DocumentProviderTypes {
```

Comment:
Use singular naming for object
```suggestion
export class DocumentProviderType {
```